subsys/bluetooth/host/hci_core.c

```diff
@@ -765,6 +765,24 @@ static void update_pending_id(struct bt_keys *keys, void *data)
 }
 #endif
 
+static struct bt_conn *find_valid_conn(bt_addr_le_t *peer_addr)
```

Comment:
This is not looking up any valid (ref-count > 0) connection, so this name is a bit misleading. Would `find_pending_connect()` be more appropriate?